api/IDBOpenDBRequest.json

```diff
@@ -137,6 +137,94 @@
           }
         }
       },
+      "blocked_event": {
+        "__compat": {
+          "description": "<code>blocked</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBOpenDBRequest/blocked_event",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "57",
+                "prefix": "webkit"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "25"
+              },
+              {
+                "version_added": "25",
+                "version_removed": "57",
+                "prefix": "webkit"
+              }
+            ],
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
+                "version_added": "10",
+                "version_removed": "16",
+                "prefix": "moz"
+              }
+            ],
+            "firefox_android": {
+              "version_added": "22"
+            },
+            "ie": {
+              "version_added": "10",
+              "notes": "partial"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "22"
+            },
+            "safari": {
+              "version_added": "7"
+            },
+            "safari_ios": {
+              "version_added": "8"
+            },
+            "samsunginternet_android": [
+              {
+                "version_added": true
+              },
+              {
+                "version_removed": "7.0",
+                "prefix": "webkit",
+                "version_added": true
```

Comment:
Seems a bit confusing for this line to be at the bottom fo this support detail; shouldn't this be above the `version_removed` line?

Reply:
This already got merged (and those things were in the original data I copied). Do you think it is worth filing a follow-up bug for them?

(In general, while I agree that your proposal is better, I think typographic fixes like this should either be enforced by the linter or made optional. See also, enforcing alphabetical listing for subfeatures.)

Reply:
I don't think it is important enough to file a followup bug, but we should perhaps consider liniting it in future.

Reply:
Filed https://github.com/mdn/browser-compat-data/issues/3505